packages/web-app-importer/public/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "entrypoint": "importer.js",
+  "config": {
+    "companionUrl": "https://host.docker.internal:9200/companion"
```

Comment:
I'd like to propose leaving the value string empty and providing the dev stack url via the `apps.yaml` file instead. Have a look at https://github.com/owncloud/web-app-jupyter/blob/main/dev/docker/ocis/apps.yaml for a working example. Otherwise the dev stack leaks into the production release.